SPECS/google-guice/google-guice.spec

```diff
@@ -0,0 +1,479 @@
+%bcond_without bootstrap
+Summary:        Lightweight dependency injection framework for Java 5 and above
+Name:           google-guice
+Version:        4.2.3
```

Comment:
why not using latest (5.1.0)? 4.2.3 is from 03/2020

Reply:
bootstrap supports 4.2.3.
The build fails with newer versions.